### ISSUE TITLE
CheckinAir Browser Incompatibility

### ISSUE THREAD

**Issue by ampatterson1** [petereit/CheckinAir#17]
When trying to print from CheckinAir/Breeze to Brother QL-810W printer from an iPad Air, we receive a "Browser Incompatibility" message. Safari is up-to-date on the iPad. The printer is configured and the wireless is working. We tested printing labels from Planning Center to the Brother printer wirelessly...thinking this may have been a wifi or printer issue. The labels printed. We can print directly from Safari to the printer wirelessly. There are no other browsers installed on the iPad. Hoping you can help!

![browserincompatibility](https://user-images.githubusercontent.com/40128181/41991809-c47e2a88-7a14-11e8-9f5b-66ea68b989ae.PNG)

**Comment by petereit** [comment]
What is the iOS version on your iPad?

**Comment by ampatterson1** [comment]
10.3.3

**Comment by petereit** [comment]
If you log into your checkin site through Safari on that iPad does it give you the same error?

**Comment by shawnsherwood** [comment]
I am having the same problem. The only problem is that when going through Safari it says that my browser is compatible. The checkinair app gives me the same error as listed above. I have tried it on multiple iPads/iPhones and they can all recognize the Brother QL-810W via the AirPrint option, but not in Checkinair or Breeze through Safari. Any help would be great! Thank you for your hard work on this project.

**Comment by petereit** [comment]
This is going to take some work on your part, but in order for me to accurately diagnose the issue, could you give me a tap-by-tap walkthrough if each action from launching the Checkin Air app through the display of the error?

**Comment by shawnsherwood** [comment]
Yes I sure will! When I get back to my office, I will screen record the process and send it to you.

Thank you!
S

**Comment by shawnsherwood** [comment]
Here you go! I may not being doing something right, lol. 

Thank you again for looking at this.
S

**Comment by petereit** [comment]
Doesn't look like your attachment came through.

**Comment by shawnsherwood** [comment]
Let’s try it again, lol! 

Shawn

**Comment by petereit** [comment]
Try sending it directly to me markpetereit @ iCloud dot com.

**Comment by petereit** [comment]
OK. Got your video. AirPrinting labels doesn't work with Breeze's web site through Safari. That's why I created my CheckinAir app.

My app hijacks the Breeze web site's printing function, recreates the labels within my app and sends them to your AirPrint label printer. But it only works if you log into Breeze through my app.

**Comment by shawnsherwood** [comment]
Okay, did you see the video of me going through the CheckinAir app and for some reason it still doesn’t recognize the printer? I am logged into the app with our Breeze login info. Am I missing a step?

Thank you!
S

**Comment by petereit** [comment]
At the current time I believe that is correct -- only users assigned to the Check In role will work in the CheckinAir app. I didn't intentionally limit the app in this way, I was simply providing a solution for my church and that's how we happen to have our system configured.

In order to get my app to work with additional roles, I will need to do a complete analysis and redesign.
